CDCgov/prime-reportstream
# Lpabon/6140 inline value sets

This PR adds support for Value Sets in the FhirToHL7 translation config. This replaces various custom functions that had been built as a workaround.

Test Steps:
1. Run the converter with the fhirdata command. e.g. `./prime fhirdata --input-file src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml --output-format=HL7`
2. Search on the config files for `valueSet` and verify that the values map correctly. For example, `OBR-25` (result status) is configured to map certain strings to single character values.

## Changes
- Added support for Value Sets on YML configs.
- Replaced custom FHIR functions that existed just for mapping with Value Sets instead.
- Added test for getting mapped values.

## Linked Issues
- #6140
